packages/web/src/pages/sign-up-page/components/layout.tsx

```diff
@@ -46,7 +48,15 @@ const transitionAxisConfig = {
 const AnimatedFlex = animated(Flex)
 
 export const Page = (props: PageProps) => {
-  const { centered, children, as, transition, transitionBack, ...other } = props
+  const {
+    centered,
+    children,
+    as,
+    transition,
+    transitionBack,
+    inputRef,
```

Comment:
nit: I would prefer we put "autofocus" in the name for this somewhere, like `autofocusRef` or `autofocusInputRef`

Reply:
Yeah great idea!!